src/components/Sidebar.stories.tsx

```diff
@@ -265,8 +273,17 @@ function Template(args: any) {
 export const Default = Template.bind({})
 
 Default.args = {
-  user: {
-    name: 'Jane Smith',
-    email: 'jane.smith@plural.sh',
-  },
+  userName: 'Jane Smith',
+  userOrganization: 'Plural',
```

Comment:
we should probably keep the naming here in line with the api, which has it as `account` not `organization`

Reply:
Sure, np